src/main.cpp

```diff
@@ -127,7 +128,7 @@ bool commandLineOptions(int argc, char *argv[])
 {
 	int opt = -1;
 
-	while((opt = getopt(argc, argv, "m:d:v:l:fh?")) != -1)
+	while((opt = getopt(argc, argv, "m:d:v:l:f:zh?")) != -1)
```

Comment:
Maybe I'm wrong, but ':' after 'f' means that we expect some parameter after '-f'.
But there is no optarg inside handler for '-f' (line 181 below).
So, maybe correct string is "m:d:v:l:fzh?"?
Also taking into account that handler is wrapped by "#ifdef WITH_ZCOPY", may be this new string should be wrapped too. Or handler should be present always but inform that ZCOPY is disabled. Maybe.

Reply:
will add 2 const strings under #ifdef WITH_ZCOPY, will remove ":"

Reply:
WRT "-f" your are rught, according to [man getopt](http://manpages.ubuntu.com/manpages/trusty/pt/man3/getopt.3.html):
`optstring is a string containing the legitimate option characters.  If such a character is followed by a colon, the option requires an argument`
Please file a bug for "-f"

Reply:
Filed a bug #92